{
  try {
    _persistence.countByE_E(RandomTestUtil.nextLong(),RandomTestUtil.nextLong());
    _persistence.countByE_E(0L,0L);
  }
 catch (  Exception e) {
    Assert.fail(e.getMessage());
  }
}
